PlayRho/Collision/DynamicTree.cpp

```diff
@@ -83,13 +83,16 @@ DynamicTree::Size RebalanceAt(DynamicTree::TreeNode nodes[], DynamicTree::Size i
     // assert(index < GetNodeCapacity());
     assert(DynamicTree::IsBranch(nodes[i].GetHeight()));
 
-    //          o
-    //          |
-    //          i
-    //         / \
-    //      *-c1  c2-*
-    //     /   |  |   \
-    // c1c1 c1c2  c2c1 c2c2
+    /*
+              o
+              |
+              i
+             / \
+          *-c1  c2-*
+         /   |  |   \
+     c1c1 c1c2  c2c1 c2c2
+
+    */
```

Comment:
I'd like to keep comment style changes out of any PR to change warning levels. I'm also not really preferring one style over the other. So this isn't really a change I'd like to take on with this PR.

Reply:
That's ok. Moreover, it is not the main standard ;)

Reply:
Ugg... I see now that g++ doesn't like multiline comments using `//` style due to `-Werror=comment`. Sigh. I'll change that style since I'd rather have warn/error free compiles that `//` style comments for these.